content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+
+{{< figure src="/docs/images/gateway-api-schema.svg" alt="gateway-api-schema" class="diagram-medium" link="https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/main/site-src/images/schema-uml.svg" >}}
+
+### GatewayClass resource
+
+Gateways can be implemented by different controllers, often with different configurations. A Gateway
+must reference a GatewayClass resource which contains the name of the controller that implements the
+class and optional configuration parameters.
+
+A minimal GatewayClass resource example:
+
+{{% code_sample file="service/networking/gateway-class.yaml" %}}
+
+In this example, a controller that implements the Gateway API is configured to manage GatewayClasses
+with the controller name `example.com/gateway-controller`. An optional `parametersRef` is specified to
+reference an implementation-specific `Parameters` custom resource containing additional configuration
+parameters. Gateways of this class will be configured with these parameters.
+
+See the [GatewayClass][gatewayclass] reference for a full definition of this resource.
+
+### Gateway resource
+
+A Gateway describes an instance of traffic handling infrastructure. It defines a network endpoint
+that can be used for processing traffic, i.e. filtering, balancing, splitting, etc. for backends
+such as a Service resource. For example, HTTP traffic sent to a Kubernetes Service by a cloud load
+balancer or an in-cluster proxy server.
+
+A minimal Gateway resource example:
+
+{{% code_sample file="service/networking/gateway.yaml" %}}
+
+In this example, an instance of traffic handling infrastructure is programmed to listen for HTTP
+traffic on port 80. Since the `addresses` field is unspecified, an address or hostname is assigned
+to the Gateway by the controller that is used as a network endpoint for processing traffic.
+
+See the [Gateway][gateway] reference for a full definition of this resource.
+
+### Route resources
+
+Route resources, e.g. HTTPRoute, describe protocol-specific rules for mapping requests from a
+Gateway resource to backend resources such as a Service. When a Route attaches to a Gateway, it
+represents configuration that is applied to the underlying Gateway implementation, e.g. load balancer.
+
+A minimal HTTPRoute resource example:
+
+{{% code_sample file="service/networking/httproute.yaml" %}}
+
+In this example, HTTP traffic from Gateway `example-gateway` with host header "www.example.com/login"
+will be routed to Service `example-svc` on port `8080`.
+
+See the [HTTPRoute][httproute] reference for a full definition of this resource.
+
+## Request flow
+
+Here is a simple example of HTTP client traffic being routed to a `Service` by using the
+Gateway and HTTPRoute resources:
+
+{{< figure src="/docs/images/gateway-api.svg" alt="gateway-api-diagram" class="diagram-medium" link="https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/main/site-src/images/httproute-basic-example.svg" >}}
```

Comment:
- You must commit the images here and serve them via our CDN. Don't rely on the `main` branch of the Gateway API having the right content.
- Please set a useful `alt` attribute.